src/dbos-executor.ts

```diff
@@ -59,11 +60,27 @@ export interface DBOSConfig {
   readonly debugProxy?: string;
 }
 
-interface WorkflowInfo<T extends any[], R> {
-  workflow: Workflow<T, R>;
+interface WorkflowInfo {
+  workflow: Workflow<any, any>;
   config: WorkflowConfig;
 }
 
+interface TransactionInfo {
+  transaction: Transaction<any, any>;
+  config: TransactionConfig;
+}
+
+interface CommunicatorInfo {
+  communicator: Communicator<any, any>;
+  config: CommunicatorConfig;
+}
```

Comment:
Should these be placed in `transaction.ts` and `communicator.ts`, respectively?

Reply:
Why?  They're not used in those files, only here